dash_renderer/__init__.py

```diff
@@ -80,5 +80,14 @@ def _set_react_version(react_version):
             '/dash_renderer/dash_renderer.min.js'
         ).format(__version__),
         'namespace': 'dash_renderer'
+    },
+    {
+        'relative_package_path': '{}.min.js.map'.format(__name__),
+        'dev_package_path': '{}.dev.js.map'.format(__name__),
+        "external_url": (
+            'https://unpkg.com/dash-renderer@{}'
+            '/dash_renderer/dash_renderer.min.js.map'
+        ).format(__version__),
+        'namespace': 'dash_renderer'
```

Comment:
Add maps to Dash

Reply:
Not sure it needs an entry here for the map, the `_js_dist` are files automatically served on the index. I think the js bundle will request the map on it's own.

Reply:
You're right that the JS file will request it on its own if the dev tools are opened -- but it will resolve it to its relative path, for example
![image](https://user-images.githubusercontent.com/8092993/49595516-1cbd6000-f946-11e8-9138-4aca54dfbde6.png)

Reply:
Or will Dash still allow the request to go through even if it's not defined here?

Reply:
In any case, I'll try it -- I assumed it was necessary :)

Reply:
Yes dash should allow it, before the map changes in dash I remember seeing 500 errors for map requests.

Reply:
Tested it out against the dash-table's examples and if the map is not present here, it doesn't get loaded. Dash probably tells the browser there is no such resource; and the browser continues on with its life?

Reply:
So it seems this additional entry is required

Reply:
Yes, but the 500 only happens *if* there is no extension support for map *and* the map is in the init file